packages/rpc/json/error.ts

```diff
@@ -0,0 +1,27 @@
+export class JsonRpcProviderError extends Error {
```

Comment:
nit: What about shorten it a bit?

```suggestion
export class JsonRpcError extends Error {
```

Reply:
[Rename to JsonRpcError](https://github.com/pilagod/waallet-extension/pull/179/commits/9e483d3864440352cfd0afc756c745d406bcc6b1)